{"metadata":{"symbolKind":"case","modules":[{"name":"MFA"}],"externalID":"s:3MFA15MFAServiceErrorO7generalyACSS_tcACmF","title":"MFAServiceError.general(message:)","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"general","kind":"identifier"},{"kind":"text","text":"("},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":")","kind":"text"}],"role":"symbol"},"abstract":[{"type":"text","text":"General error with custom message."}],"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/mfa\/mfaserviceerror\/general(message:)"]}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"general","kind":"identifier"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError\/general(message:)"},"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"references":{"doc://com.ibm.verify.ios.sdk/documentation/MFA":{"url":"\/documentation\/mfa","role":"collection","title":"MFA","abstract":[],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA","kind":"symbol","type":"topic"},"doc://com.ibm.verify.ios.sdk/documentation/MFA/MFAServiceError/general(message:)":{"type":"topic","title":"MFAServiceError.general(message:)","abstract":[{"text":"General error with custom message.","type":"text"}],"identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError\/general(message:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"kind":"symbol","role":"symbol","url":"\/documentation\/mfa\/mfaserviceerror\/general(message:)"},"doc://com.ibm.verify.ios.sdk/documentation/MFA/MFAServiceError":{"abstract":[],"kind":"symbol","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/MFA\/MFAServiceError","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"MFAServiceError","kind":"identifier"}],"navigatorTitle":[{"text":"MFAServiceError","kind":"identifier"}],"title":"MFAServiceError","url":"\/documentation\/mfa\/mfaserviceerror","type":"topic"}}}